content/tokio/tutorial/async.md

```diff
@@ -558,18 +573,38 @@ impl MiniTokio {
     }
 }
 
+impl TaskFuture {
+    fn new(future: impl Future<Output = ()> + Send + 'static) -> TaskFuture {
+        TaskFuture {
+            future: Box::pin(future),
+            poll: Poll::Pending,
+        }
+    }
+
+    fn poll(&mut self, cx: &mut Context<'_>) {
+        // Spurious wake-ups are allowed, even after a future has                                  
```

Comment:
it might be worth linking to wherever in the `std::future` documentation it explains that the future trait contract allows spurious wakeups?

Reply:
I'm not actually sure where that's documented, and didn't find anything with a quick search in the obvious candidates. It may be implied by other documentation, or just "common knowledge". 

I only became aware of it because @Darksonn [commented such on a previous PR](https://github.com/tokio-rs/website/pull/739#pullrequestreview-1777531251).

Reply:
@hawkw any thoughts on the above 👆? Otherwise this is good to go 🚀